website/docs/reference/rbac.md

```diff
@@ -82,7 +82,7 @@ You can assign the following project permissions. The permissions will be valid
 
 - **create/edit variants**
 
-  Lets the user create and edit variants within the project.
+  Lets the user create and edit variants within the project. (Deprecated with v4.21 in favor of environment-specific permissions for working with variants)
```

Comment:
Can you add a footnote here?

```suggestion
  Lets the user create and edit variants within the project. (Deprecated with v4.21 in favor of environment-specific permissions for working with variants[^1].)
```

And then something like this at the end of the document?
```
[^1]: The project-level permission is still required for the [**create/overwrite variants** (PUT)](api/unleash/overwriteFeatureVariants.api.mdx) and [**update variants** (PATCH)](api/unleash/patchFeatureVariants.api.mdx) API endpoints, but it is not used for anything within the admin UI. The API endpoints have been superseded by the [**create/overwrite environment variants** (PUT)](api/unleash/overwrite-environment-feature-variants.api.mdx) and [**update environment variants** (PATCH)](api/unleash/patch-environment-feature-variants.api.mdx) endpoints, respectively.
```

Reply:
Yes, on it!